packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -173,12 +167,8 @@ export function BlockSettingsDropdown( {
 		]
 	);
 
-	const label = sprintf(
-		/* translators: %s: block name */
-		__( 'Remove %s' ),
-		blockTitle
-	);
-	const removeBlockLabel = count === 1 ? label : __( 'Remove blocks' );
+	const removeBlockLabel =
+		count === 1 ? __( 'Delete' ) : __( 'Delete blocks' );
```

Comment:
I can understand removing the block title for the single block deletion. For multiple block deletion, how do you expect non-sighted users to have a reference point on the blocks they are deleting? You get this context visually by some type of selected styling. These are things we should at least think about when deleting multiple blocks, it might not be 100% clear to users like it would be with 1 block. I for one did not know the block toolbar was available for more than 1 selected block. I thought it was only available on a block-by-block bases.

Reply:
This is a valid point, but I don't know that it needs to block _this_ PR? The flow for removing multiple blocks hasn't changed apart from the label.

In terms of a solution, how would you expect a flow like this to work?

Reply:
You are correct. I think we need to handle this at the toolbar level, not the item-specific label. Could add a description to the toolbar if that is possible via ARIA to let users know about the currently selected blocks.

Reply:
Agreed. I'm not changing the communication on multi-block deletion, other than use the word "Delete" consistently. But yes we can certainly improve on that front. Thanks for the review and thoughts—much appreciated.